tokio-util/src/codec/length_delimited.rs

```diff
@@ -777,6 +795,42 @@ impl Builder {
         self
     }
 
+    /// Sets the unsigned integer type used to represent the length field.
+    ///
+    /// The default type is [`u32`]. The max type is [`u64`] (or [`usize`] on
+    /// 64-bit targets).
```

Comment:
The example uses `u128`?

Reply:
The compilation failure example uses `u128`.